### PR TITLE
Add Goerli + roles

### PR DESCRIPTION
**Pull request by akegaviar**
Added Goerli.
Added roles.
Cleaned up the Polygon bridge tutorial + added Goerli.

### CODE REVIEW

File: docs/platform/invite-a-user-to-the-organization.md [hunk 1]
```diff
@@ -8,18 +8,21 @@ meta:
 
 # Invite a user to the organization
 
-You can invite a [user](/glossary/user) to your [organization](/glossary/organization) who will:
+You can invite a [user](/glossary/user) to your [organization](/glossary/organization) and assign one of the following roles:
 
-* Create both public and consortium projects.
-* Deploy nodes and networks.
-* Access all existing projects, networks, nodes, identities and their access and credentials.
+* Admin — the role can invite users, create and delete projects, deploy and delete nodes and networks, change user roles.
```
Comment (easeev) [inline docs/platform/invite-a-user-to-the-organization.md:13]: >change user roles

I don't think it's possible

You should check but AFAIK the only real difference between Admin and Editor is that Admin has access to billing details/settings and Editor hasn't

File: docs/platform/invite-a-user-to-the-organization.md [hunk 2]
```diff
@@ -8,18 +8,21 @@ meta:
...
+* Editor — the role can create and delete projects, deploy and delete nodes and networks.
```
Comment (easeev) [inline docs/platform/invite-a-user-to-the-organization.md:14]: AFAIK can invite/delete users in the org but we should double check

**Review by easeev — CHANGES_REQUESTED** [review]

File: docs/platform/invite-a-user-to-the-organization.md (hunk 1 above)
Comment (akegaviar) [reply docs/platform/invite-a-user-to-the-organization.md:13]: Checked with Andrey, what's currently in the PR is correct. Admin can change roles, Editor cannot change roles.

File: docs/platform/invite-a-user-to-the-organization.md (hunk 2 above)
Comment (akegaviar) [reply docs/platform/invite-a-user-to-the-organization.md:14]: Checked with Andrey, Editor cannot invite/delete users.

File: docs/platform/invite-a-user-to-the-organization.md (hunk 1 above)
Comment (easeev) [reply docs/platform/invite-a-user-to-the-organization.md:13]: Ok, what about billing?

Comment (easeev) [reply docs/platform/invite-a-user-to-the-organization.md:13]: BTW who can show me how to change role in the UI? I can't figure out how to do it

Comment (akegaviar) [reply docs/platform/invite-a-user-to-the-organization.md:13]: Fixed

File: docs/release-notes.md [hunk 3]
```diff
@@ -8,6 +8,21 @@ meta:
...
+## Chainstack 2.11
+
+*April 14, 2022*
+
+### What’s new
+
+* **Protocols**. [Ethereum Goerli](/operations/ethereum/networks) support.
+* **Role management**
+  * You can now [assign roles](/platform/invite-a-user-to-the-organization) to your organization's users.
+  * There are now three roles: <a href="https://support.chainstack.com/hc/en-us/articles/900001563563-Organizations-users-and-members" target="_blank">Admin, Editor, Viewer</a>. Only the Admin role has access to the organization's billing settings.
```
Comment (easeev) [inline docs/release-notes.md:20]:
```suggestion
  * There are now three roles: <a href="https://support.chainstack.com/hc/en-us/articles/900001563563-Organizations-users-and-members" target="_blank">Admin, Editor, Viewer</a>.
```

**Pull request merged**